client/components/CarouselModal.jsx

```diff
@@ -72,7 +70,15 @@ class CarouselModal extends Component {
       toggleModal,
     } = this.props;
     const { xHover, counter, modalMeasurement } = this.state;
-    const dots = urls.map((c, index) => <td key={c}><a href={`#slide-${index}`}><div className={index === currentIndex ? 'selectedDot' : 'unSelectedDot'}><InlineIcon icon={dotIcon} onClick={this.pickFromDot} data-index={index} className={index === currentIndex ? 'bigDot' : 'dot'} color="gray" height={index === currentIndex ? '20' : '20'} width={index === currentIndex ? '20' : '20'} /></div></a></td>);
+    const dots = urls.map((c, index) => (
+      <td key={c}>
+        <a href={`#slide-${index}`}>
+          <div className={index === currentIndex ? 'selectedDot' : 'unSelectedDot'}>
+            <InlineIcon icon={dotIcon} onClick={this.pickFromDot} data-index={index} className={index === currentIndex ? 'bigDot' : 'dot'} color="gray" height="20" width="20" />
+          </div>
+        </a>
+      </td>
+    ));
```

Comment:
Thank you! My eyes were bleeding. I'd still pull all the InlineIcon props down into individual lines - I think this still violates the convention of 80 characters per line max as best practices.

Reply:
Good call, thanks Robert!